### ISSUE TITLE
empty yaml file crashes the controller

### ISSUE THREAD

**Issue by gaya-** [SemRoCo/giskard_examples#38]

**Comment by airballking** [comment]
Indeed, I am right now not catching invalid yaml commands. Two things need to happen:
- `whole_body_controller`: catch the corresponding exception and finish the goal callback gracefully
- `controller_action_server`: check for invalid yamls yourself

The former is straight-forward, the latter might take more time. I have to think about what to do..
